test/jdk/javax/sound/sampled/Clip/DataPusherThreadCheck.java

```diff
@@ -70,7 +70,7 @@ public static void main(String[] args) throws Exception {
     }
 
     private static void checkThread(File file) throws Exception {
-        AudioClip clip = (AudioClip) file.toURL().getContent();
+        SoundClip clip = SoundClip.createSoundClip(file);
```

Comment:
I think we should either support content handlers via the new SoundClip object or remove the existing implementation, as it is currently tied to AudioClip.

Reply:
Not sure I would do that anyway here. It seems like an unnecessary file->url->file roundtrip with a cast to a type you have to "know". And it was never specified.